challenges/2022-10-11-blast-from-the-past/solutions/csharp/mrumiker/mrumiker.cs

```diff
@@ -0,0 +1,31 @@
+using Xunit;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public class BlastfromthePast
+{
+  public int[] FindThreeLargestNums(int[] input)
+
+  {
+    var inputLength = input.Length;
+    if (inputLength < 3) return new int[] { };
+
+    var output = new int[3];
+    Array.Copy(input, output, 3); // copy the first three numbers in our array into our output array
+    Array.Sort(output);
+    for (int i = 3; i < inputLength; i++)
+    {
+      var currentNum = input[i];
+      var currentLow = output[0];
+      if (currentNum > currentLow)
```

Comment:
nice little trick here!

Reply:
I like it! It's a (very tiny) _bit_ less efficient than the optimal solution, but I feel like the readability really makes up for it.